src/Elastic/Elastic.Autorest/docs/Get-AzElasticOrganizationApiKey.md

```diff
@@ -24,16 +24,16 @@ Get-AzElasticOrganizationApiKey -Body <IUserEmailId> [-SubscriptionId <String[]>
  [-Confirm] [-WhatIf] [<CommonParameters>]
 ```
 
-### GetViaIdentity
+### GetViaJsonFilePath
```

Comment:
This cmdlet is actually a post request with request body. Please follow our best practice to update the directive.